packages/guides/src/Compiler/Passes/GlobalMenuPass.php

```diff
@@ -118,7 +121,7 @@ private function getMenuEntryWithChildren(CompilerContext $compilerContext, Menu
     private function addSubEntries(
         CompilerContext $compilerContext,
         MenuEntryNode $sectionMenuEntry,
-        DocumentEntryNode $documentEntry,
+        DocumentEntryNode|ExternalEntryNode $entryNode,
```

Comment:
We can change this to `EntryNode` as argument, as both classes extend this node type. Stretching the accepted types isn't a breaking change in a concreet implementation. like this class, especially because it's final.

Reply:
Using EntryNode I had trouble with PHP-stan that I could not resolve...

Reply:
also we are explicitly handling these two types while there could be others...?

Reply:
What was the error?